test/performance/perf_test.lua

```diff
@@ -101,15 +101,19 @@ local row_name = {
     insert = 'insert',
```

Comment:
It seems, `dev_checks()` is not disabled for performance tests.

Reply:
Well, it is really crucial and I had completely forgot about it. Thank you for your comment. Now results seems more optimistic